inst/templates/common/header.mustache

```diff
@@ -10,7 +10,9 @@
 <div class="header-labels">
 <div class="header-col-left">
 <p><strong>Location:</strong> {{reportMetadata.stationId}} - {{reportMetadata.stationName}}</p>
+<!-- Only DVHydro and 5YR GW use primarySeriesLabel -->
 {{#reportMetadata.primarySeriesLabel}}<p><strong>Primary Time Series:</strong> {{reportMetadata.primarySeriesLabel}}</p>{{/reportMetadata.primarySeriesLabel}}
+{{#reportMetadata.timeseriesLabel}}<p><strong>Primary Time Series:</strong> {{reportMetadata.timeseriesLabel}}</p>{{/reportMetadata.timeseriesLabel}}
```

Comment:
Instead of adding to header I'd fix on the java side to be consistent with other reports. I think "primarySeriesLabel" is the most widely used.